sktime/datatypes/_alignment/_examples.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""Example generation for testing.
+
+Exports dict of examples, useful for testing as fixtures.
+
+example_dict: dict indexed by triple
+  1st element = mtype - str
+  2nd element = considered as this scitype - str
+  3rd element = int - index of example
+elements are data objects, considered examples for the mtype
+    all examples with same index are considered "same" on scitype content
+    if None, indicates that representation is not possible
+"""
+
+import pandas as pd
+
+example_dict = dict()
+
+###
```

Comment:
?

Reply:
separator for different fixtures/examples.
There's just one in the file so it might be confusing.

As just discussed on slack, would leave it in for consistency.